### ISSUE TITLE
change robot tests to robotframework browser libray test, remove selenium based tests

### ISSUE THREAD

**Issue by 1letter** [plone/bobtemplates.plone#572]
since the plone 6 core use only playwright based robot tests, the templates should be updated also

**Comment by MrTango** [comment]
I agree, but i would like to see how the native Playwright integration plays out, as this would be so much easier to work with than writing robot tests. So my energy i spend first there, but I'm not opposed to this update. It would be good to have.
I'll work on bobtemplates.plone and plonecli modernization at the Bucharest Sprint.

**Comment by MrTango** [comment]
@1letter i have it working in https://github.com/MrTango/collective.pyplaywrightdemo.
@gforcada and his team will also work on that for there project.
I'll try to improve it on the 21.03. (tune up friday).
Main thing to do, is to add an autologin feature like plone.app.robotframework has, to plone.app.testing, so everybody can use it in there projects. Also we might come up with some useful pytest fixtures over time, similar to what we have in Plone core keywords for robot tests.

**Comment by MrTango** [comment]
closing this in favor of #583
